src/client/localplayer.cpp

```diff
@@ -583,16 +582,8 @@ void LocalPlayer::applyControl(float dtime, Environment *env)
 		}
 	}
 
-	if (continuous_forward)
-		speedH += v3f(0,0,1);
-
 	if (control.up) {
-		if (continuous_forward) {
-			if (fast_move)
-				superspeed = true;
```

Comment:
why do we lost this behaviour ?

Reply:
It's not lost. `aux1` is set to `true` if `up` is pressed with activated autoforward. (See [here](https://github.com/minetest/minetest/pull/8249/files#diff-7f71a92de20504e8a5636b4317ea67b2R2489).)

Reply:
it's a little bit hacky, no ? aux1 is a key there is no key to be pressed for superspeed?

Reply:
Well, normally aux1(E) is pressed for superspeed and sprint mods for example will probably use this to determine whether a player wants to sprint. I don't think that this is more hacky than setting the up key on autoforward.

Reply:
So, what changes do you request?

Reply:
@nerzhul any comment?

Reply:
I'm unresolving this because it needs consideration, due to my other line comment and because:
Autoforward is a new feature i added, when i added it i wasn't aware of how it combined with 'up' and 'fast move', if i had been aware i would have probably reconsidered the existing behaviour. I think the existing behaviour needs reconsideration now.

Reply:
Change of mind: I'm ok with removing the feature of 'autoforward' + 'up' = 'fast', as it is a duplicated control. See my other line comment for details.

Reply:
I think i misunderstood. It seems to me that superspeed does need to be set to true when 'up' is pressed while 'autoforward' is active.

Reply:
SmallJoker agrees.